comps/comps-pulpcore-el8.xml

```diff
@@ -25,6 +25,7 @@
       <packagereq type="default">pulpcore-selinux</packagereq>
       <packagereq type="default">python3-createrepo_c</packagereq>
       <packagereq type="default">python3-libcomps</packagereq>
+      <packagereq type="default">python3-pulp-cli-deb</packagereq>
```

Comment:
Since `pulp-cli` is added under `<packagereq type="default">python38-pulp-cli</packagereq>` I am thinking it should be:
```suggestion
      <packagereq type="default">python38-pulp-cli-deb</packagereq>
```
and then rearranged to alphabetical order?